maven-javadoc-plugin/pom.xml

```diff
@@ -193,9 +193,9 @@ under the License.
 
     <!-- misc -->
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.6</version>
```

Comment:
The plugin is compatible with Java 6, but Commons Lang 3.6 requires Java 7. We can only upgrade to 3.5 for now.